packages/server-ct/src/routes-ct.ts

```diff
@@ -77,7 +77,7 @@ export const createRoutes = ({
     xhrs.handle(req, res, config, next)
   })
 
-  app.get(config.clientRoute, (req, res) => {
+  app.get(`${config.clientRoute}`, (req, res) => {
```

Comment:
If this is wrapped b/c for type reasons, should we explicitly handle it rather than just assuming coercion will be correct? Not sure the specifics, so this may be fine.

Reply:
Defensive is good... fixed: https://github.com/cypress-io/cypress/pull/17200/commits/2fc9fc1a49fdc563f6a48b1b7af8752fc15e550d